.storybook/foundations/corner-rounding/corner-rounding.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Canvas, Story, Title } from '@storybook/blocks';
+import * as CornerRoundingStories from './corner-rounding.stories.js';
+import ComponentSizeTokensSmall from './component-size-tokens-small.mdx';
+import ComponentSizeTokensMedium from './component-size-tokens-medium.mdx';
+import * as CheckboxCornerRoundingStories from './checkbox-corner-rounding.stories.js';
+import * as ActionButtonCornerRoundingStories from './action-button-corner-rounding.stories.js';
+
+<Meta of={CornerRoundingStories} />
+
+<Title />
+
+## Alias tokens
+
+<Story of={CornerRoundingStories.CornerRounding} />
+
+## Component size tokens
+
+In addition to the alias tokens listed above, Spectrum 2 also provides component size alias tokens that align with global size tokens:
+
+### Small component size tokens
+
+<ComponentSizeTokensSmall />
+
+### Medium component size tokens
+
+<ComponentSizeTokensMedium />
+
+These component size tokens are used to scale corner rounding in different-sized tokens. For instance, the corner-rounding for a checkbox will be `4px` for a medium-sized checkbox, `3px` for small, `5px` for large, and `6px` for an extra-large checkbox, corresponding to the small component size tokens table above.
```

Comment:
While we use the phrase "t-shirt size" internally, is this common for the public facing docs? I know in the components docs and Storybook it is just called "Sizes" or "Sizing", and I see some component Guidelines pages for Spectrum 1 that just say "Sizes" as well. I'm not sure if Spectrum 2 is making more use of that terminology or not.

Also, I'd try to keep "extra large" instead of "x-large" where possible, to be extra clear when considering screen readers or readers less fluent in english.

Reply:
Great points, thank you!